src/utilities/process-readme.js

```diff
@@ -1,5 +1,11 @@
 const url = require('url');
 
+const beginsWithDocsDomainRegex = /^https?:\/\/webpack\.js\.org/;
+
+const redirects = {
```

Comment:
Uhmm, I was thinking of using webpack redirects instead of this string replacement
https://github.com/webpack/webpack.js.org/blob/rebuild/webpack.prod.js#L60

Reply:
Those would not be evaluated when doing file based link checking I think?

Reply:
My idea is to put setup the redirect, that plugin creates meta tags with the redirect ()https://github.com/bebraw/redirect-webpack-plugin/blob/master/index.js#L15), so when hyperlink hit that link it would be redirected.

Reply:
That might work. I'm pretty sure hyperlink models meta redirects. But it might show the redirect as an error and indicate that you should update the source url to the redirect target. We can probably mark that check as `--todo` or `--skip` though

Reply:
![image](https://user-images.githubusercontent.com/331790/44335359-79b70800-a474-11e8-8f3d-c72bec9f5795.png)

![image](https://user-images.githubusercontent.com/331790/44335368-84719d00-a474-11e8-85b6-9b8586b519ab.png)

Reply:
No PRs, just ask the maintainer to push a commit updating that README.

Reply:
Project README updated. I'll remove the rewriting code. Though some of the refactorings will stay, since I discovered that fragment links had the original source prepended, so some of it actually fixes a bug